src/vivarium_gates_lsff/components/__init__.py

```diff
@@ -1,2 +1,3 @@
 from .disease import VitaminADeficiency
 from .disease import IronDeficiency
+from .disease import NeonatalSWC_without_incidence
```

Comment:
What does SWC stand for?

Reply:
I actually don't know... It is a public health default model:
`vivarium_public_health.disease.models.NeonatalSWC_without_incidence`